### DISCUSSION TITLE
No Page "transitions" in Android

### DISCUSSION THREAD

**Discussion by munkii** [PrismLibrary/Prism#3296] [.NET MAUI]
Most of our navigations are non modal. In iOS i see the simple "Page Slide In" transition and the opposite when going back. With Android we see no page transitions in or out for non modal navigation.

Modal navigation has an animated page transition on both iOS and Android.

I have tried the PrismMauiDemo app and that also does not have page transitions for non modal. Is this a MAUI issue or something particular to Prism?

Any example navigation path with PrismMauiDemo is 

1. Flyout Page
2. Go Back View B
3. View C
4. View D
5. Go Back to Root
6. Hamburger Menu, Select a Page

I do not see any page transitions doing any of that Navigation with Android, except when I hack about with the code so that on Step 6 we load a page Modally

**Comment by AmrAlSayed0** [comment]
This could be a phone setting. Animations could be disabled from Android's Developer Options.

**Reply by dansiegel** [reply]
have you tried explicitly specifying the animated property in the NavigationParameters?

**Reply by munkii** [reply]
Yes, I explicitly set modal and animated.

```
protected async Task NavigateAsync<T>(object data = null, bool modal = false, bool animate = true)
    where T : BreatheViewModel
{
    var navigationParams = new NavigationParameters();

    if (data != null)
    {
        navigationParams.Add("initData", data);
    }

    if (animate == true)
    {
        navigationParams.Add(KnownNavigationParameters.Animated, true);
    }

    if (modal == true)
    {
        navigationParams.Add(KnownNavigationParameters.UseModalNavigation, true);
    }

    await this.NavigationService.NavigateAsync(typeof(T).Name, navigationParams);
}
```

I took the out of the box MS MAUI template "with Sample Content" (the Task list app)  that uses Shell Navigation and have started to hack Prism into it as a repro rather than using Shell Nav. 

I am seeing the same problem on Android devices but not on Apple. I'll get something posted after christmas
